src/utils/Terminal.php

```diff
@@ -210,6 +210,7 @@ public static function toANSI(string $string) : string{
 				TextFormat::LIGHT_PURPLE => Terminal::$COLOR_LIGHT_PURPLE,
 				TextFormat::YELLOW => Terminal::$COLOR_YELLOW,
 				TextFormat::WHITE => Terminal::$COLOR_WHITE,
+				TextFormat::MINECOIN_GOLD => Terminal::$COLOR_YELLOW,
```

Comment:
This shouldn't reuse the YELLOW property. If plugins want to make it display as a separate colour, they should be able to do so.